resources/conf/config.toml.template

```diff
@@ -183,10 +183,9 @@ retainKeys = ["self_validate_jwt", "issuer", "claim_mappings", "consumer_key_cla
   # Maximum value that can be set as the count within retry configs in an OpenAPI definition or API-M UI
   maxRetryCount = 5
   # Base interval for the Envoy's exponential retry back off algorithm
-  baseIntervalInMillis = 25
-  # HTTP status codes, the retry mechanism will be enabled for.
-  # Used when retry config is set via API-M UI or all given status codes are out of range.
-  statusCodes = [ 504 ]
```

Comment:
statusCodes is missing in the template file

Reply:
StatusCode we won't be utilizing in choreo.

Reply:
But it is there in the `adapter/config/types.go` file.